### DISCUSSION TITLE
文件列表中无法查看隐藏文件

### DISCUSSION THREAD

**Discussion by Linya-IronMan** [nshen/learn-neovim-lua#61] [Q&A] [answered]
有些环境配置文件命名是以 .env.develpoment  这样来命名的，这样的话在文件列表中就无法看到。
命令行中可以通过 `ls -a` 查看到这些文件，
ctrl + F 查找文件内容也无法对隐藏的文件内容进行检索

**Comment by allworldg** [comment]
https://waylonwalker.com/til/nvim-telescope-hidden-files/试一下这个

**Reply by Linya-IronMan** [reply]
不知道怎么插入到 telescope 的配置中

**Comment by Linya-IronMan** [comment, marked as answer]
在 plugin-config/nvim-tree.lua 文件中有配置，filters ，打开就行了
<img width="477" alt="image" src="https://user-images.githubusercontent.com/77786952/185569959-22e41b5f-6759-48be-add5-87e7883fb1de.png">

**Comment by nshen** [comment]
我设置的快捷键 `.` 显示和隐藏 点文件

**Reply by nshen** [reply]
i 是显示隐藏 gitignore的文件
